orientdb/README.md

```diff
@@ -42,16 +42,19 @@ See the next section for the list of configuration parameters for OrientDB.
 ## OrientDB Configuration Parameters
 
 * ```orientdb.url``` - (required) The address to your database.
-    * Supported storage types: memory, plocal
+    * Supported storage types: memory, plocal, remote
     * EX. ```plocal:/path/to/database```
 * ```orientdb.user``` - The user to connect to the database with.
     * Default: ```admin```
 * ```orientdb.password``` - The password to connect to the database with.
     * Default: ```admin```
-* ```orientdb.newdb``` - Create the database if it does not exists, or overwrite the database if it does exist.
+* ```orientdb.newdb``` - Overwrite the database if it already exists.
```

Comment:
So, there really is NO reason this should be set during the run phase, that I can think of. It will always kill the workload because the database will be empty.

Would adding a note of "Only effects load phase", and then only honoring this setting during the load phase be acceptable?